controllers/resources.go

```diff
@@ -19,6 +19,10 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+var (
+	affineRegex = regexp.MustCompile("\\n\\s+specialresource\\.openshift\\.io\\/kernel\\-affine")
```

Comment:
How about using backticks here for a raw string? That should avoid some escaping.